### DISCUSSION TITLE
Folder name specific colors

### DISCUSSION THREAD

**Discussion by WilliamWelsh** [nvim-tree/nvim-tree.lua#1498] [Q&A]
Is it possible to change the folder color, and maybe even icon, based on the name of the folder? For example, all of my folders are grey, but I want to make folders named `node_modules` green, folders named `lua` blue, folders named `pages` orange, etc.

**Comment by alex-courtis** [comment]
No. That's a really good idea.

I imagine you could create a highlight group similar to `NvimTreeImageFile`, say, `NvimTreeLuaFolder`

Please raise a [feature request](https://github.com/kyazdani42/nvim-tree.lua/issues/new?assignees=&labels=feature+request&template=feature_request.md&title=). Please include a mockup of what to include in setup to achieve this functionality.
